build.gradle

```diff
@@ -127,6 +127,14 @@ apiValidation {
             'stream-chat-android-test',
             'stream-chat-android-compose-sample',
             'stream-chat-android-ui-guides',
+            'stream-chat-android-client-metrics-baseline',
+            'stream-chat-android-client-metrics-stream',
+            'stream-chat-android-offline-metrics-baseline',
+            'stream-chat-android-offline-metrics-stream',
+            'stream-chat-android-ui-components-metrics-baseline',
+            'stream-chat-android-ui-components-metrics-stream',
+            'stream-chat-android-compose-metrics-baseline',
+            'stream-chat-android-compose-metrics-stream',
```

Comment:
Can't we use only one gradle project with multiple flavors??

Reply:
Good point, I will check that, thanks 🙌🏻